packages/loader/src/SpriteAtlasLoader.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  resourceLoader,
+  Loader,
+  AssetPromise,
+  AssetType,
+  LoadItem,
+  ResourceManager,
+  Texture2D,
+  Sprite,
+  SpriteAtlas
+} from "@oasis-engine/core";
+import { AtlasConfig } from "@oasis-engine/core/types/2d/atlas/types";
+import { Rect, Vector2 } from "@oasis-engine/math";
+
+@resourceLoader(AssetType.SpriteAtlas, ["atlas"], false)
+class SpriteAtlasLoader extends Loader<SpriteAtlas> {
+  load(item: LoadItem, resourceManager: ResourceManager): AssetPromise<SpriteAtlas> {
+    return new AssetPromise((resolve, reject) => {
+      this.request<AtlasConfig>(item.url, {
+        ...item,
+        type: "json"
+      })
+        .then((atlasData) => {
+          const atlasItems = atlasData.AtlasItems;
+          const atlasItemsLen = atlasItems.length;
+          const picsArr: string[] = new Array(atlasItemsLen);
```

Comment:
Use  `const picsArr = new Array<string>(atlasItemsLen);` instead of `const picsArr: string[] = new Array(atlasItemsLen);`